packages/itinerary-body/src/AccessLegBody/index.tsx

```diff
@@ -17,7 +17,11 @@ import TNCLeg from "./tnc-leg";
 import { defaultMessages } from "../util";
 
 interface Props {
-  config: Config;
+  config: Config & {
+    itinerary?: {
+      hideDrivingDirections?: boolean;
+    };
+  };
```

Comment:
I think you should add this to the `config` type in OTP-UI rather than adding it here so it's centralized!

Reply:
@binh-dam-ibigroup what do you think? You're adding typescript to the config so maybe this point is mute

Reply:
Spoke with Binh about this offline, and I think we'll deal with config typing in subsequent PRs if that's alright!

Reply:
At some point, I would actually prefer to remove passing a config object and pass individual props instead. (That would be a breaking change)